dt-core/admin/menu/tabs/tab-fields.php

```diff
@@ -102,26 +102,52 @@ public function box_message( $type ) {
 
             <table class="widefat striped">
                 <tr>
-                    <th style="width:30%">Name</th>
+                    <th style="width:5%"></th>
+                    <th style="width:20%">Name</th>
                     <th style="width:20%">Key</th>
                     <th style="width:10%">Type</th>
+                    <th style="width:5%; text-align: center;">Icon</th>
                     <th style="width:40%">Details</th>
                 </tr>
             <?php
             foreach ( $fields as $field_key => $field_value ){
                 if ( $type === $field_value["type"] ){
                     ?>
                     <tr>
+                        <td>
+                            <form method="get">
+                                <input type="hidden" name="field_select_nonce" id="field_select_nonce" value="<?php echo esc_attr( wp_create_nonce( 'field_select' ) ) ?>" />
+                                <input type="hidden" name="page" value="dt_options" />
+                                <input type="hidden" name="tab" value="custom-fields" />
+                                <input type="hidden" name="post_type" value="<?php echo esc_attr( $post_type ); ?>" />
+                                <input type="hidden" name="field-select" value="<?php echo esc_html( $post_type . '_' . $field_key ) ?>" />
+
+                                <button type="submit" class="button" name="field_selected"><?php esc_html_e( 'Edit', 'disciple-tools' ) ?></button>
+                            </form>
+                        </td>
                         <td><?php echo esc_html( $field_value["name"] ) ?></td>
                         <td><?php echo esc_html( $field_key ) ?></td>
                         <td><?php echo esc_html( $field_value["type"] ) ?></td>
+                        <td align="center">
```

Comment:
`align="centre"` flagged as an obsolete attribute within my dev-env; maybe refactor to use `style="text-align: center;"`

Overall, logic seems to be holding true; with `Edit` button redirecting to corresponding field details view.

Good Job!! 👍

Reply:
sweet, thanks. Yeah it was showing up as red in my vscode but didn't tell me off. I'll change it over...

Reply:
@kodinkat ok, that should do it